src/components/Menu/Menu.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { MenuItem } from "../../components";
+
+export class Menu extends React.Component {
+  render() {
+    const { sandwiches, breads, addToOrder } = this.props;
+
+    return (
+      <div className="menu">
+        <h1>Menu</h1>
+        {sandwiches.map((sandwich, i) => {
+          const { name, price } = sandwich;
+          return (
+            <MenuItem
```

Comment:
When the name and values are the same, you can use this syntax. Not sure if I like it less or more though? Might be overkill dude

```js
<MenuItem
  key={i}
  {...{ name }}
  {...{ price }}
  {...{ addToOrder }}
  {...{ breads }}
/>
```

Reply:
That looks ugly as hell doesn't It haha